snapcraft/internal/sources/_subversion.py

```diff
@@ -53,14 +59,37 @@ def pull(self):
 
         if os.path.exists(os.path.join(self.source_dir, '.svn')):
             subprocess.check_call(
-                [self.command, 'update'] + opts, cwd=self.source_dir)
+                [self.command, 'update'] + opts, cwd=self.source_dir,
+                **self._call_kwargs)
         else:
             if os.path.isdir(self.source):
                 subprocess.check_call(
                     [self.command, 'checkout',
                      'file://{}'.format(os.path.abspath(self.source)),
-                     self.source_dir] + opts)
+                     self.source_dir] + opts, **self._call_kwargs)
             else:
                 subprocess.check_call(
                     [self.command, 'checkout', self.source, self.source_dir] +
-                    opts)
+                    opts, **self._call_kwargs)
+
+        self.source_details = self._get_source_details()
+
+    def _get_source_details(self):
+        branch = None
+        tag = None
+        source = self.source
+        commit = self.source_commit
+
+        lines = subprocess.check_output(['svn', 'info', self.source_dir]
+                                        ).decode('utf-8').split('\n')
+        prefix = 'Last Changed Rev: '
+        for line in lines:
+            if line.startswith(prefix):
+                commit = line.replace(prefix, '')
```

Comment:
This variable may have already been set, in which case you're throwing it away here. In the other sources you check to make sure that isn't the case. Is there a reason that's not happening here?